web/modules/custom/blog/blog.info.yml

```diff
@@ -0,0 +1,7 @@
+name: 'Blog'
+type: module
+description: 'for blog'
```

Comment:
Clearly state what this module was created for, for example: 'Provide blog blocks for the main page'